am-lib/src/test/java/uk/gov/hmcts/reform/amlib/DefaultRoleSetupImportServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.reform.amlib;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.amlib.enums.AccessManagementType;
+import uk.gov.hmcts.reform.amlib.enums.RoleType;
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_NAME;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_TYPE;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.SERVICE_NAME;
+
+class DefaultRoleSetupImportServiceTest {
+    private final DefaultRoleSetupImportService defaultRoleSetupImportService = new DefaultRoleSetupImportService(
+        "", "", "");
+
+    @Test
+    void whenServiceNameNullThrowNullPointerException() {
+        assertThatExceptionOfType(NullPointerException.class)
+            .isThrownBy(() -> defaultRoleSetupImportService.addService(null, ""));
+    }
+
+    @Test
+    void whenServiceNameIsEmptyStringThrowIllegalArgumentException() {
+        assertThatExceptionOfType(IllegalArgumentException.class)
+            .isThrownBy(() -> defaultRoleSetupImportService.addService("", ""))
+            .withMessage("Service name cannot be empty");
+    }
+
+    @Test
+    void whenRoleNameIsNullThrowNullPointerException() {
+        assertThatExceptionOfType(NullPointerException.class)
+            .isThrownBy(() -> defaultRoleSetupImportService.addRole(
+                null, RoleType.RESOURCE, SecurityClassification.Public, AccessManagementType.ROLEBASED));
+    }
+
+    @Test
+    void whenRoleNameIsEmptyStringThrowIllegalArgumentException() {
```

Comment:
We should include method name in test name or split into separate classes otherwise we will have problem with duplicates.

I sort of prefer separate classes as it give us one place to rename when method name changes.